src/game/client/gui/guicallbacks/controlbarcallback.cpp

```diff
@@ -12,18 +12,360 @@
  *            A full copy of the GNU General Public License can be found in
  *            LICENSE
  */
+#include "animatewindowmanager.h"
+#include "commandxlat.h"
 #include "controlbar.h"
+#include "diplomacy.h"
+#include "display.h"
+#include "gadgettextentry.h"
+#include "gameclient.h"
+#include "globaldata.h"
+#include "languagefilter.h"
+#include "mouse.h"
+#include "player.h"
+#include "playerlist.h"
+#include "quitmenu.h"
+#include "radar.h"
+#include "scriptengine.h"
+#include "view.h"
 
-void Hide_Control_Bar(bool hide)
+void Hide_Control_Bar(bool immediate)
```

Comment:
Perhaps `bool fully`, because there are 2 modes for Control Bar hide:

Button toggle: hides Control Bar a bit.

![shot_20231107_200012_1](https://github.com/TheAssemblyArmada/Thyme/assets/4720891/100cb925-ceaa-4883-84a8-98ee0de99380)

F9 key: hides Control Bar fully.

![shot_20231107_200016_2](https://github.com/TheAssemblyArmada/Thyme/assets/4720891/e176c4d4-13f0-4d62-912a-540b3c1a660f)

Reply:
This is correct name per original.